### DISCUSSION TITLE
Import on cloud works, import on self-hosted not

### DISCUSSION THREAD

**Discussion by SaschaValentin** [Budibase/budibase#12257] [Help] [answered]
I´ve seen a similar issue on a few other posts, but this one seems to be a bit different.

When I import a former exported Budibase app on my self-hosted version "Error creating app" appears - but only on this app. Importing other ones work fine. Then I´ve tested to import this app on the cloud version of Budibase and this works without problems.

Can anyone give me a hint where to look for what is wrong with the self-hosted version? I have even tried to export the app from the cloud version ones more and import it in my hosted one, but it still doesn´t work.

I have no idea what I can do but I´m really dispaired because I´ve worked for over three months on this app and I was just a few steps before completing it. Hope someone has an idea what I can do.

**Comment by ConorWebb96** [comment]
Hey @SaschaValentin,

- Are there any network request errors or console errors that may provide a bit more information on this?
- What data source is this app using?

Before trying this make sure you have a backup. Can you remove all data from the app and then try and import that into the self-host instance? If this works, this would suggest to me that theres likely and issue somewhere with your data.

**Comment by SaschaValentin** [comment, marked as answer]
Hi @ConorWebb96,

I´ve reinstalled Budibase this morning using the latest version and it worked. Yesterday somehow it seems that I had installed a former version. So everything works and I´m really relieved.

Anyway, thanks for your help.
